src/Makefile.am

```diff
@@ -1197,6 +1197,12 @@ install-headers:
 	for header in $(noinst_HEADERS); do \
```

Comment:
"rust-bindings.h was not being installed with rust-bindings.h, and its" doesn't seem to make much sense to me.